examples/react-native/README.md

```diff
@@ -0,0 +1,13 @@
+# Usage with React Native
+
+React Native has no `document` context, this means you need to specify the
+reporter in the configuration. For example:
```

Comment:
I don't understand this without context. Probably should be: you should specify `reporter: xhr`, because default reporter uses `document` which is not available in React.